dashboard/client/src/components/LogView/LogVirtualView.tsx

```diff
@@ -118,8 +189,25 @@ const LogVirtualView: React.FC<LogVirtualViewProps> = ({
   if (listRef) {
     listRef.current = outter.current;
   }
+  const [selectedLogLine, setSelectedLogLine] = useState<string>();
+  const handleLogLineClick = useCallback((logLine: string) => {
+    setSelectedLogLine(logLine);
+  }, []);
+
   const itemRenderer = ({ index, style }: { index: number; style: any }) => {
     const { i, origin } = logs[revert ? logs.length - 1 - index : index];
+
+    let message = origin;
+    let formattedLogLine = origin;
+    try {
+      const parsed_origin = JSON.parse(origin);
```

Comment:
```suggestion
      const parsedOrigin = JSON.parse(origin);
```
nit: idiomatic typescript uses camelCase

Reply:
oh right, thanks for catching it!